feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/data/network/subscan/request/StakingSumRewardRequest.kt

```diff
@@ -0,0 +1,11 @@
+package jp.co.soramitsu.feature_staking_impl.data.network.subscan.request
+
+class StakingSumRewardRequest(accountAddress: String) {
+    val query = """
+        |{
```

Comment:
Without this it will look like {"query":"\n        {\n        sumReward\n           (id: \"H12456789012345678901567890124567\")\n           {accountTotal}\n        }\n        "}

Or the formatting will have to start at the first column of a file

Reply:
Done